packages/blade/scripts/manualPureFunctions.js

```diff
@@ -0,0 +1,19 @@
+const pureFunctions = ['assignWithoutSideEffects'];
+
+/**
+ * This plugin marks defined function name as PURE. This is temporary implementation of manualPureFunctions.
+ *
+ * Once we upgrade to rollup 3.5.0, we can use treeshake.manualPureFunctions config from rollup instead of this plugin.
+ * https://rollupjs.org/configuration-options/#treeshake-manualpurefunctions
+ */
+const manualPureFunctions = () => ({
```

Comment:
:p why is this "manual", shouldn't this be "auto"?

Reply:
Can't we upgrade our rollup version and use this?

Reply:
we can. It's a major version bump though so didn't spend much time right now. can pick it up next quarter